spec/models/case_cycle_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CaseCycle, type: :model do
+  let(:invalid_date_msg) { 'Invalid date' }
+
+  it { should validate_numericality_of(:copay).is_greater_than(0) }
+
+  it 'validates uniqueness of slug' do
+    create(:case_cycle)
+    should validate_uniqueness_of(:slug)
+  end
+
+  it 'validates submitted_on date if present' do
+    cycle = build(:case_cycle, submitted_on: nil)
+    expect(cycle).to be_valid
+
```

Comment:
I would add lines to explicitly show that setting `cycle.submitted` to a valid `Date` is valid (`be_valid`).  That will provide an example/specification that shows that a `Date` is the correct thing to use.   (And the same for `effective_on`, `expires_on`, and  `notified_on`